README.md

```diff
@@ -125,3 +125,5 @@ There are several automated fuzzing tests in fwupd. These take some time to run:
     ninja fuzz-synaptics-rmi
     ninja fuzz-firmware
     ninja fuzz-smbios
+
+
```

Comment:
why did you change this?

Reply:
Change in number of Hypens below "Other frontends".
This change had gone by mistake in last merge. And Richard had reverted this in his commit. I did accept this change from Richard.